ymlフォルダ/VPC.yml

```
@@ -0,0 +1,145 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Network_Layer Template
+
+Resources:
+  SampleVPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.0.0.0/16
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      InstanceTenancy: default
+      Tags:
+        - Key: name
+          Value: SampleVPC
+  PublicSubnet1a:
+    Type: AWS::EC2::Subnet
+    Properties:
+      VpcId: !Ref SampleVPC
+      CidrBlock: 10.0.0.0/24
+      MapPublicIpOnLaunch: true
+      AvailabilityZone: ap-northeast-1a
+      Tags:
+      - Key: Name
+        Value: PublicSubnet1a

```

何用のパブリックサブネットかがわかる名前が望ましいです。
実務ではシステム名の短縮語を入れたりします。
```suggestion
      - Key: Name
        Value: SamplePublicSubnet1a

```

---

PublicSubnet1aに変更しました！

---

私のコメントと対応内容がマッチしていませんので再度コメントを確認お願いします！

---

RaiseTechPublicSubnet1aにValueの名称を変更しました。